WooCommerce/WooCommerceTests/ViewRelated/Products/Media/ProductImagesProviderTests.swift

```diff
@@ -0,0 +1,74 @@
+import Photos
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class ProductImagesProviderTests: XCTestCase {
```

Comment:
After some debugging, it seems like `PHImageManager`'s init triggered the permission alert. If you run a test case that has no `PHImageManager` in it, and then add a `let imageManager = PHImageManager()`, the alert appears afterward.

To avoid having to create an instance of `PHImageManager` in unit tests, I created a protocol `PHAssetImageLoader` for `PHImageManager`: https://github.com/woocommerce/woocommerce-ios/pull/1878/commits/f574d0aa8cd26708c6f4b0a56b3fc515c06fc958